src/main/java/it/gov/pagopa/arc/controller/NoticesControllerImpl.java

```diff
@@ -0,0 +1,32 @@
+package it.gov.pagopa.arc.controller;
+
+import it.gov.pagopa.arc.controller.generated.ArcNoticesApi;
+import it.gov.pagopa.arc.dto.NoticesListResponseDTO;
+import it.gov.pagopa.arc.model.generated.NoticesListDTO;
+import it.gov.pagopa.arc.service.NoticesService;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class NoticesControllerImpl implements ArcNoticesApi {
+    private final NoticesService noticesService;
+
+    public NoticesControllerImpl(NoticesService noticesService) {
+        this.noticesService = noticesService;
+    }
+
+    @Override
+    public ResponseEntity<NoticesListDTO> getNoticesList(String xContinuationToken, Integer size, Boolean paidByMe, Boolean registeredToMe, String orderBy, String ordering) {
+        NoticesListResponseDTO noticesListResponseDTO = noticesService.retrieveNoticesAndToken(xContinuationToken, size, paidByMe, registeredToMe, orderBy, ordering);
+
+        String continuationToken = StringUtils.isNotBlank(noticesListResponseDTO.getContinuationToken()) ? noticesListResponseDTO.getContinuationToken() : null;
```

Comment:
an if statement would be more readable than ternary operator.
you could set directly the header value, without the need of the variable